shub/config.py

```diff
@@ -16,6 +16,8 @@
                         pwd_hg_version, pwd_git_version, pwd_version)
 
 
+SH_IMAGES_PREFIX = 'images.scrapinghub.com/project/'
+SH_IMAGES_REGISTRY = SH_IMAGES_PREFIX + '{project}'
```

Comment:
btw, I'd also add some check for correctness of sh_images path before all of `shub image` commands (maybe even in config) - I accidentally put repository name with wrong project id (valid project I have access to, but different project id from what it should deploy to)  and it worked :/ 

<img width="1428" alt="1__alex_mbp____dev_workspace_scrapinghub_repo_testspiders__zsh_" src="https://cloud.githubusercontent.com/assets/744331/24907870/e5ab4076-1eb5-11e7-9a59-926c3b1eac81.png">